### PR TITLE
Add leap wallet data

### PR DESCRIPTION
**Pull request by baryon2**

### CODE REVIEW

File: wallets/leapextension.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "irisnet",
+    "jackal",
+    "kujira",
+    "likecoin",
+    "mars",
```
Comment (JeremyParish69) [inline wallets/leapextension.json:30]: mars not registered to chain registry. Is it live yet?

File: wallets/leapextension.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "persistence",
+    "atlantic",
```
Comment (JeremyParish69) [inline wallets/leapextension.json:32]: atlantic not registered to chain registry

File: wallets/leapextension.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "sifchain",
+    "sommelier",
+    "stargaze",
+    "starname",
+    "stride",
+    "teritori",
+    "umee"
+  ],
+  "features": ["suggest_chain"],
```
Comment (JeremyParish69) [inline wallets/leapextension.json:41]: Do you have any documentation on the suggest chain feature? Would like to learn more about how users can add chain to their wallet

Comment (baryon2) [reply wallets/leapextension.json:41]: Here is the link to the suggest chain docs.
https://docs.leapwallet.io/cosmos/connect-to-leap-on-dapps/suggest-chain

File: wallets/leapextension.json (hunk 1 above)
Comment (baryon2) [reply wallets/leapextension.json:30]: It is live on testnet. Let me know if this list should only include mainnet chains, I'll remove it from the list.

File: wallets/leapextension.json (hunk 2 above)
Comment (baryon2) [reply wallets/leapextension.json:32]: atlantic is registered on the [testnet registry](https://github.com/cosmos/chain-registry/tree/master/testnets/atlantic). Let me know if this list is supposed to include only mainnet chains, I'll update the pr accordingly.

Comment (JeremyParish69) [reply wallets/leapextension.json:32]: You are correct, 'atlantic' is registered to the chain registry as a testnet. At the moment, it is undetermined how exactly we'll handle testnets in the Wallet Registry. I'm inclined to temporarily allow this to be included until the decision is made.

File: wallets/leapextension.json (hunk 1 above)
Comment (JeremyParish69) [reply wallets/leapextension.json:30]: Although we haven't decided yet on whether/how to include testnets herein, I feel quite confident that we likely won't allow chains that aren't yet registered to the chain registry--and neither 'mars' nor 'marstestnet' are registered yet. Thus, I'd like to ask that this be removed for now.

Comment (baryon2) [reply wallets/leapextension.json]: Ok, I have removed mars from the list.

**Pull request merged**